app/_data/docs_nav_kic_1.0.x.yml

```diff
@@ -280,162 +280,6 @@
-          url: /deployment/gke
-        - text: Admission Controller
-          url: /deployment/admission-webhook
-    - text: Guides
-      items:
-        - text: Overview
-          url: /guides/overview
-        - text: Getting Started with KIC
-          url: /guides/getting-started
-        - text: Getting Started using Istio
-          url: /guides/getting-started-istio
-        - text: Using Custom Resources
-          items:
-            - text: Using the KongPlugin Resource
-              url: /guides/using-kongplugin-resource
-            - text: Using the KongIngress Resource
-              url: /guides/using-kongingress-resource
-            - text: Using KongConsumer and KongCredential Resources
-              url: /guides/using-consumer-credential-resource
-            - text: Using the KongClusterPlugin Resource
-              url: /guides/using-kongclusterplugin-resource
-        - text: Using the ACL and JWT Plugins
-          url: /guides/configure-acl-plugin
-        - text: Using cert-manager with Kong
-          url: /guides/cert-manager
-        - text: Configuring a Fallback Service
-          url: /guides/configuring-fallback-service
-        - text: Using an External Service
-          url: /guides/using-external-service
-        - text: Configuring HTTPS Redirects for Services
-          url: /guides/configuring-https-redirect
-        - text: Using Redis for Rate Limiting
-          url: /guides/redis-rate-limiting
-        - text: Integrate KIC with Prometheus/Grafana
-          url: /guides/prometheus-grafana
-        - text: Configuring Circuit-Breaker and Health-Checking
-          url: /guides/configuring-health-checks
-        - text: Setting up a Custom Plugin
-          url: /guides/setting-up-custom-plugins
-        - text: Using Ingress with gRPC
-          url: /guides/using-ingress-with-grpc
-        - text: Setting up Upstream mTLS
-          url: /guides/upstream-mtls
-        - text: Exposing a TCP-based Service
-          url: /guides/using-tcpingress
-        - text: Using the mTLS Auth Plugin
-          url: /guides/using-mtls-auth-plugin
-        - text: Configuring Custom Entities
-          url: /guides/configuring-custom-entities
-        - text: Using the OpenID Connect Plugin
-          url: /guides/using-oidc-plugin
-        - text: Rewriting Hosts and Paths
-          url: /guides/using-rewrites
-        - text: Preserving Client IP Address
-          url: /guides/preserve-client-ip
-    - text: References
-      items:
-        - text: KIC Annotations
-          url: /references/annotations
-        - text: CLI Arguments
-          url: /references/cli-arguments
-        - text: Custom Resource Definitions
-          url: /references/custom-resources
-        - text: Plugin Compatibility
-          url: /references/plugin-compatibility
-        - text: Version Compatibility
-          url: /references/version-compatibility
-        - text: Troubleshooting
-          url: /troubleshooting
-
```

Comment:
For the `kic` nav files, it should be the other way around: leave this portion, delete everything else.